### ISSUE TITLE
grid scenario warnings

### ISSUE THREAD

**Issue by AboudyKreidieh** [flow-project/flow#44]
Any idea why this is happening?
```
Warning: At actuated tlLogic 'center0', minDur 3.00 is too short to short for detector gap of 28.00m.
Warning: At actuated tlLogic 'center1', minDur 3.00 is too short to short for detector gap of 28.00m.
Warning: At actuated tlLogic 'center2', minDur 3.00 is too short to short for detector gap of 28.00m.
Warning: At actuated tlLogic 'center3', minDur 3.00 is too short to short for detector gap of 28.00m.
Warning: At actuated tlLogic 'center4', minDur 3.00 is too short to short for detector gap of 28.00m.
Warning: At actuated tlLogic 'center5', minDur 3.00 is too short to short for detector gap of 28.00m.
Warning: At actuated tlLogic 'center6', minDur 3.00 is too short to short for detector gap of 28.00m.
Warning: At actuated tlLogic 'center7', minDur 3.00 is too short to short for detector gap of 28.00m.
Warning: At actuated tlLogic 'center8', minDur 3.00 is too short to short for detector gap of 28.00m.
Warning: At actuated tlLogic 'center9', minDur 3.00 is too short to short for detector gap of 28.00m.
Warning: At actuated tlLogic 'center10', minDur 3.00 is too short to short for detector gap of 28.00m.
Warning: At actuated tlLogic 'center11', minDur 3.00 is too short to short for detector gap of 28.00m.
Warning: At actuated tlLogic 'center12', minDur 3.00 is too short to short for detector gap of 28.00m.
Warning: At actuated tlLogic 'center13', minDur 3.00 is too short to short for detector gap of 28.00m.
Warning: At actuated tlLogic 'center14', minDur 3.00 is too short to short for detector gap of 28.00m.
Warning: At actuated tlLogic 'center15', minDur 3.00 is too short to short for detector gap of 28.00m.
Warning: At actuated tlLogic 'center16', minDur 3.00 is too short to short for detector gap of 28.00m.
Warning: At actuated tlLogic 'center17', minDur 3.00 is too short to short for detector gap of 28.00m.
Warning: At actuated tlLogic 'center18', minDur 3.00 is too short to short for detector gap of 28.00m.
Warning: At actuated tlLogic 'center19', minDur 3.00 is too short to short for detector gap of 28.00m.
Warning: At actuated tlLogic 'center20', minDur 3.00 is too short to short for detector gap of 28.00m.
Warning: At actuated tlLogic 'center21', minDur 3.00 is too short to short for detector gap of 28.00m.
Warning: At actuated tlLogic 'center22', minDur 3.00 is too short to short for detector gap of 28.00m.
Warning: At actuated tlLogic 'center23', minDur 3.00 is too short to short for detector gap of 28.00m.
Warning: At actuated tlLogic 'center24', minDur 3.00 is too short to short for detector gap of 28.00m.
```

**Comment by kjang96** [comment]
It looks like this is occurring because the minimum duration of a traffic light is shorter than the amount of time it takes a vehicle to get from the detector gap to the intersection. A detector gap could detect a gap, change the phase (with the aim of getting the last detected vehicle through the intersection), and the phase switch occurs before the last detected vehicles gets to/through the intersection.

**Comment by kjang96** [comment]
On second thought I think this might be a Sumo bug, because in the grid.py experiment, where the parameters are set to speed_limit = 35m/s, detector_gap = 0.8 s, and minDur = 3s, you can evidently clear 0.8s in 3s

**Comment by eugenevinitsky** [comment]
You can clear it, but you won't necessarily. If you increase min_dur the warning goes away.

**Comment by AboudyKreidieh** [comment]
resolved by #211
